.gitignore

```diff
@@ -1,6 +1,7 @@
 pkg/*
 *.gem
 *.sw*
+.idea
```

Comment:
This belongs in your user-level .gitignore that sits in your home directory (`~/.gitignore`). Not in this project.